app/decorators/assignment_repo_decorator.rb

```diff
@@ -1,12 +1,12 @@
 class AssignmentRepoDecorator < Draper::Decorator
   delegate_all
 
-  def avatar_url(size)
-    "https://avatars.githubusercontent.com/u/#{assignment_repo_user.uid}?v=3&size=#{size}"
-  end
+  delegate :full_name, to: :github_repository
 
-  def full_name
-    github_repository.full_name
+  AVATAR_BASE_URL = 'https://avatars.githubusercontent.com/u/'
```

Comment:
just a proposal for avoiding Magic String, it can be used in the tests, is reusable and makes the line a bit shorter